apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.service.ts

```diff
@@ -170,7 +171,22 @@ export const include: Includeable[] = [
   },
   { model: Case, as: 'parentCase', attributes },
   { model: Case, as: 'childCase', attributes },
-  { model: Defendant, as: 'defendants' },
+  {
+    model: Defendant,
+    as: 'defendants',
+    required: false,
+    order: [['created', 'ASC']],
+    include: [
```

Comment:
Why is this needed in limitedAccessCaseService? Do defenders need to be able to see subpoenas as well?

Reply:
Yes, they need to see subpoenas.